tools/generate-record-row.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
```

Comment:
Suggest changing this to python3 since `python` is ambigious and not available on newer linux distros (ubuntu 20.04)

Reply:
ack deferring to py3 build env migration